ldap/servers/slapd/slapi-private.h

```diff
@@ -1260,8 +1260,8 @@ void dup_ldif_line(struct berval *copy, const char *line, const char *endline);
 
 /* slapi-memberof.c */
 int slapi_memberof(Slapi_MemberOfConfig *config, Slapi_DN *member_sdn, Slapi_MemberOfResult *result);
-void slapi_memberof_free_memberof_plugin_config();
-int slapi_memberof_load_memberof_plugin_config();
+void slapi_memberof_free_memberof_plugin_config(void);
```

Comment:
Should we also change https://github.com/389ds/389-ds-base/blob/18ef874d839036e5f025acf511c03dbc57ccef70/ldap/servers/slapd/slapi-memberof.c#L583 ? (and the same for other function)